charts/prod-ohstaff-values.yaml

```diff
@@ -78,6 +79,11 @@ persistentVolumes:
     server: "lib-partners.in.library.ucla.edu"
     path: "/OralHistory"
   ohmasters:
+    storage: "7Ti"
```

Comment:
Similar to the above, I expect to see both `ohmasters` and `ohmasterslz` placeholders in the generic `values.yaml`.